ykman/driver_ccid.py

```diff
@@ -300,12 +300,12 @@ def kill_scdaemon():
                 killed = True
     except ImportError:
         # Works for Linux and OS X.
-        pids = subprocess.check_output(
+        pids = subprocess.check_output(  # nosec
             "ps ax | grep scdaemon | grep -v grep | awk '{ print $1 }'",
             shell=True).strip()
         if pids:
             for pid in pids.split():
-                subprocess.call(['kill', '-9', pid])
+                subprocess.call(['kill', '-9', pid])  # nosec
```

Comment:
I think it would be better to replace both of these subprocess calls with one:

subprocess.call(["/usr/bin/pkill", "-9", "scdaemon"])                                                                                                   

This works on macOS and on Linux, /usr/bin/pkill is the correct path on macOS, OpenBSD, FreeBSD, ArchLinux, Debian, Ubuntu and Fedora (yes, I checked all of them now).  

This reduced the number of warnings from bandit from 5 to 2.

Reply:
Agreed, fixed.